trustmanager/keys.go

```diff
@@ -9,6 +9,7 @@ import (
 	"sort"
 	"strings"
 
+	"fmt"
```

Comment:
super nit: let's move this to the block above with the other standard golang imports

Reply:
fixed